Tests/SourceKitLSPTests/RewriteSourceKitPlaceholdersTests.swift

```diff
@@ -0,0 +1,66 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SKTestSupport
+@testable import SourceKitLSP
```

Comment:
Shoot, I missed this other `@testable` import last night. Does this need to be changed as well @ahoppen ?

Reply:
Yes, we can’t have `@testable` imports.

Reply:
Fixed!